modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -0,0 +1,383 @@
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib;
+
+import com.liferay.frontend.taglib.clay.internal.servlet.taglib.BaseContainerTag;
+import com.liferay.frontend.taglib.clay.servlet.taglib.soy.HorizontalCard;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.taglib.util.TagResourceBundleUtil;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+
+/**
+ * @author Marko Cikos
+ */
+public class HorizontalCardTag extends BaseContainerTag {
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		Map<String, String> data = getData();
+
+		if (data != null) {
+			for (Map.Entry<String, String> entry : data.entrySet()) {
+				setDynamicAttribute(
+					StringPool.BLANK, "data-" + entry.getKey(),
+					entry.getValue());
+			}
+		}
+
+		return super.doStartTag();
+	}
+
+	public List<DropdownItem> getActionDropdownItems() {
+		if ((_actionDropdownItems != null) && (_horizontalCard != null)) {
+			return _horizontalCard.getActionDropdownItems();
+		}
+
+		return _actionDropdownItems;
+	}
+
+	@Override
+	public String getCssClass() {
+		if ((super.getCssClass() == null) && (_horizontalCard != null)) {
+			if (_horizontalCard.getCssClass() != null) {
+				return _horizontalCard.getCssClass();
+			}
+
+			if (_horizontalCard.getElementClasses() != null) {
+				return _horizontalCard.getElementClasses();
+			}
+		}
+
+		return super.getCssClass();
+	}
+
+	public Map<String, String> getData() {
+		if ((_data == null) && (_horizontalCard != null)) {
+			return _horizontalCard.getData();
+		}
+
+		return _data;
+	}
+
+	public String getHref() {
+		if ((_href == null) && (_horizontalCard != null)) {
+			return _horizontalCard.getHref();
+		}
+
+		return _href;
+	}
+
+	public String getIcon() {
+		if (_icon == null) {
+			if (_horizontalCard == null) {
+				return "folder";
+			}
+
+			return _horizontalCard.getIcon();
+		}
+
+		return _icon;
+	}
+
+	@Override
+	public String getId() {
+		if ((super.getId() == null) && (_horizontalCard != null)) {
+			return _horizontalCard.getId();
+		}
+
+		return super.getId();
+	}
+
+	public Boolean getInteractive() {
+		if ((_interactive == null) && (_horizontalCard != null)) {
```

Comment:
Not sure about this either, but didn't we say we shouldn't consider the `interactive` attribute?

Reply:
Yes, I will remove everything related to `interactive`.